src/controllers/searchProduct.js

```diff
@@ -0,0 +1,27 @@
+const { products } = require("../sequelized/models");
+const {Op} = require("sequelize")
+
+const searchProducts = async (req, res) => {
+  const page = Number(req.query.page) || 1;
```

Comment:
No need for the controller. 
This Use-case should be handled in getProducts controller.